src/drivers/imx/esai.c

```diff
@@ -281,9 +295,9 @@ static void esai_stop(struct dai *dai, int direction)
 {
 	direction = direction == SOF_IPC_STREAM_PLAYBACK;
 	/* disable exception interrupt */
-	dai_update_bits(dai, REG_ESAI_xCR(direction), ESAI_xCR_xEIE_MASK, 0);
+	dai_update_bits(dai, REG_ESAI_xCR(direction), ESAI_xCR_xEIE, 0);
```

Comment:
This looks like some sort of cleanup. Can you please create a separate patch. Or am I missing something/

Reply:
This one I will actually squash as it is introduced by the initial commit in this set.

Reply:
@paulstelian97 thanks for doing this! So, lets only have 'squashed' commits sent for review with everyone. This makes things easier to review and less confusing :)